### PR TITLE
chore: #MAG-87 add readme/licence + eventStore

### PR DESCRIPTION
**Pull request by Sofianebernoussi**
## Describe your changes
- add README
- add LICENCE
- add Event store

## Issue ticket number and link

https://jira.support-ent.fr/browse/MAG-87

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

**Review by flmartineau — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,163 @@
-# magneto
\ No newline at end of file
+# À propos de l'application Magneto
+
+* Licence : [AGPL v3](http://www.gnu.org/licenses/agpl.txt) - Copyright Ville de Paris, Région Nouvelle Aquitaine, Région Hauts de France
```
Comment (sjollois) [inline README.md:3]: pas de région idf ?

File: README.md [hunk 2]
```diff
@@ -1 +1,163 @@
...
+        "ssl" : $ssl,
+        "auto-redeploy": false,
+        "userbook-host": "${host}",
+        "integration-mode" : "HTTP",
+        "app-registry.port" : 8012,
+        "mode" : "${mode}",
+        "entcore.port" : 8009,
+        "websocket-config": {
+            "wsPort": 4404,
+            "endpoint-proxy": "/magneto/eventbus"
```
Comment (sjollois) [inline README.md:32]: c'est nouveau ça ?

File: README.md [hunk 3]
```diff
@@ -1 +1,163 @@
...
+
+
+## Documentation
+Magneto est un outil de création permettant aux utilisateurs de créer et d’échanger des tableaux. 
+Il contient des aimants, chaque aimant ayant son propre type (texte, image, lien etc..).
+
+# Modèle de données - base MongoDB
+Deux collections sont utilisées :
+"magneto.boards" : un document représente un tableau
+"magneto.cards" : un document représente un aimant
```
Comment (sjollois) [inline README.md:54]: rajouter un retour à la ligne

**Review by sjollois — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Sofianebernoussi) [reply README.md:3]: pas pour le moment a priori

File: README.md (hunk 2 above)
Comment (Sofianebernoussi) [reply README.md:32]: c'est pour le collaboratif

**Pull request merged**